doozerlib/cli/__main__.py

```diff
@@ -205,15 +203,12 @@ def validate_semver_major_minor_patch(ctx, param, version):
     vsplit = version.split(".")
     try:
         int(vsplit[0].lstrip('v'))
-        minor_version = int('0' if len(vsplit) < 2 else vsplit[1])
-        patch_version = int('0' if len(vsplit) < 3 else vsplit[2])
+        int('0' if len(vsplit) < 2 else vsplit[1])
```

Comment:
This will allow minor version to be omitted. Is this intended?

Reply:
It seems unlikely we'll use that. Just thought some other user of doozer might appreciate it someday.